src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,21 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao = new ActorDaoImpl(HibernateUtil.getSessionFactory());
```

Comment:
The service layer should not be responsible for creating DAO instances. Instead, consider using dependency injection to provide the DAO to the service. This can be done by adding a constructor to `ActorServiceImpl` that accepts an `ActorDao` as a parameter and using that to set `actorDao`. This will make your code more modular and easier to test.